### PR TITLE
Chore remove modals

### PR DESCRIPTION
**Pull request by chrisspang**

### CODE REVIEW

File: globals/z-index.css [hunk 1]
```diff
@@ -9,6 +9,8 @@
   --z-8: 80;
   --z-9: 90;
 
+  --z-modalCover:      var(--z-7);
+
```
Comment (johnPhillips) [inline globals/z-index.css]: ...do we need this?

**Comment by johnPhillips** [comment]
LGTM

File: globals/z-index.css (hunk 1 above)
Comment (chrisspang) [inline globals/z-index.css]: YES!  It's used in the Cover component:

```
components/Cover/Cover.css:  z-index: var(--z-modalCover);
```

**Comment by apacala** [comment]
:+1:

**Pull request merged**